src/commandSocket.cpp

```diff
@@ -707,20 +765,19 @@ void CommandSocket::PathCompiler::flushPathSegments()
 }
 
 //TODO: Reason about these functions when points is empty or the polygon entered is empty
-void CommandSocket::PathCompiler::sendLine(PrintFeatureType print_feature_type, Point from, Point to, int width)
+void CommandSocket::PathCompiler::sendLineTo(PrintFeatureType print_feature_type, Point to, int width)
 {
-    handleInitialPoint( from );
+    assert( points.size() > 0 );
```

Comment:
When doing an assert, it would be nice to see what the assertion means.
One simple way to do this is like:
```
assert(points.size() > 0 && "Sending a line to a new location presupposes we already have a current location set!");
```